cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -782,6 +813,7 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
                     {...dynamcPopoverPros}
                     placement='right'
                     overlayClassName='cvat-tools-control-popover'
+                    mouseLeaveDelay={1}
```

Comment:
Should we add such delay to label fields in all tools also?

Reply:
It is reasonable. Just there weren't such the request